flang/unittests/Optimizer/Builder/ComplexHelperTest.cpp

```diff
@@ -0,0 +1,64 @@
+//===- ComplexExprHelperTest.cpp -- ComplexExprHelper unit tests ----------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "flang/Optimizer/Builder/Complex.h"
+#include "gtest/gtest.h"
+#include "flang/Optimizer/Builder/FIRBuilder.h"
+#include "flang/Optimizer/Support/InitFIR.h"
+#include "flang/Optimizer/Support/KindMapping.h"
+
+struct ComplexHelperTest : public testing::Test {
+public:
+  void SetUp() override {
+    fir::KindMapping kindMap(&context,
+        "i10:80,l3:24,a1:8,r54:Double,c20:X86_FP80,r11:PPC_FP128,"
+        "r12:FP128,r13:X86_FP80,r14:Double,r15:Float,r16:Half,r23:BFloat");
```

Comment:
Why not use `fromDefaultKinds` from [https://github.com/flang-compiler/f18-llvm-project/blob/fir-dev/flang/include/flang/Optimizer/Support/Utils.h#L28-L41](Utils.h)?

Reply:
Good point. And this seems to be the function that is called by default. I have removed the custom kindmap setting.

Reply:
Testing the non-defaults is not a bad idea though.